Block Mobilism Market & others

MrApocalypse opened the pull request:
This app is the mobile app of Mobilism, that contain lots of pirated apps and themes, both free & paid versions. That's no bueno!!

MrApocalypse committed:
Block Mobilism Market

This app is the mobile app of Mobilism, that contain lots of pirated apps and themes, both free & paid versions. That's no bueno!!

AlmightyMegadeth00 commented:
Thank you. Will check it out after this weekend
On Aug 1, 2015 12:06 PM, "Corey Edwards" notifications@github.com wrote:

> This app is the mobile app of Mobilism, that contain lots of pirated apps
> 
> ## and themes, both free & paid versions. That's no bueno!!
> 
> You can view, comment on, or merge this pull request online at:
> 
>   https://github.com/AlmightyMegadeth00/AntiPiracySupport/pull/8
> Commit Summary
> - Block Mobilism Market
> 
> File Changes
> - _M_ src/utils/AntiPiracyConstants.java
>   https://github.com/AlmightyMegadeth00/AntiPiracySupport/pull/8/files#diff-0
>   (3)
> 
> Patch Links:
> - https://github.com/AlmightyMegadeth00/AntiPiracySupport/pull/8.patch
> - https://github.com/AlmightyMegadeth00/AntiPiracySupport/pull/8.diff
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/AlmightyMegadeth00/AntiPiracySupport/pull/8.

MrApocalypse committed:
Block cracked Google Play installler

There are a bunch of them, but this is one is the most commonly used. These installers allow the user to install a cracked Google Play that can bypass license verifications, leading to pirated apps/features.

There are more, I'll add once I find their package names.

AlmightyMegadeth00 committed:
AntiPiracyInstallreceiver: Move the toast out of package check loop

AlmightyMegadeth00 committed:
Add Z market to the list

AlmightyMegadeth00 committed:
Update readme

AlmightyMegadeth00 committed:
Add cracked play store that gives refunds without uninstalling

AlmightyMegadeth00 committed:
Add mobilism market

AlmightyMegadeth00 committed:
Update readme (required commits list)

MrApocalypse committed:
Merge remote-tracking branch 'contentguard/master' into lp-mr1
